CHANGES.md

```diff
@@ -3,9 +3,12 @@ Change Log
 
 ### 1.42 - 2017-02-01
 
-* Added optional scene request render mode to reduce CPU usage
-    * `scene.requestRenderMode` enables a mode which will only request new render frames on changes to the scene, or when the simulation time change exceeds `scene.maximumRenderTimeChange`.
+* Breaking changes
+    * `scene.preRender` and `scene.postRender` events are called immediately before and after scene rendering only if the scene renders a frame. See `scene.requestRenderMode`. Use `scene.preUpdate` and `scene.postUpdate` for task that require regular updates.
```

Comment:
I actually don't think this needs to be a breaking change. Though the connection between render and update events still needs to be clear. Maybe below write:

> Added `scene.preUpdate` and `scene.postUpdate` events that are raised before and after the scene updates respectively. The scene is always updated before executing a potential render. Continue to listen to `scene.preRender` and `scene.postRender` for when the scene renders a frame.